sympy/integrals/tests/test_transforms.py

```diff
@@ -618,11 +619,14 @@ def sinc(x):
     # NOTE: the ift comes out in pieces
     assert IFT(1/(a + 2*pi*I*x), x, posk,
             noconds=False) == (exp(-a*posk), True)
-    assert IFT(1/(a + 2*pi*I*x), x, -posk,
-            noconds=False) == (0, True)
+    # Boundary value implementation assumes that x and k are symbols.
```

Comment:
Why doesn't this work?

Reply:
-posk is not a symbol. That can be worked around if necessary by replacing it by a symbol during computation.

Reply:
But I didn't think it was needed as in most practical cases it is a symbol. Technically, the reason why it does not work is that it is used as an argument of a derivative.

Reply:
Oh I see you take `diff(x)` in the polylog `_eval_at_reals`. 

I don't see why `IFT` should allow expressions as symbols, but if it is allowed, perhaps it should work with a dummy symbol and replace the expression at the end. 

Also, if `at_reals` requires a symbol that should be documented and checked. 

Or is it possible to remove the `diff(x)`? I don't understand why that is there.